src/status_im/ui/screens/events.cljs

```diff
@@ -103,13 +103,17 @@
  (fn [{:keys [db]} [_ path v]]
    {:db (assoc-in db path v)}))
 
+(fx/defn on-return-from-background [cofx]
+  (fx/merge cofx
+            (inbox/request-messages)
+            (hardwallet/return-back-from-nfc-settings)))
+
 (defn app-state-change [state {:keys [db] :as cofx}]
   (let [app-coming-from-background? (= state "active")]
-    (handlers-macro/merge-fx cofx
-                             {::app-state-change-fx state
-                              :db                   (assoc db :app-state state)}
-                             (inbox/request-messages app-coming-from-background?)
-                             (hardwallet/return-back-from-nfc-settings app-coming-from-background?))))
+    (fx/merge cofx
+              {::app-state-change-fx state
+               :db                   (assoc db :app-state state)}
+              (on-return-from-background))))
```

Comment:
`app-coming-from-background?` is not used, is it expected?

Reply:
thanks good catch